vita3k/gui/src/settings_dialog.cpp

```diff
@@ -750,13 +750,13 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Save color surfaces to files.");
         ImGui::Spacing();
-        ImGui::Checkbox("Dump textures", &host.cfg.dump_textures);
+        ImGui::Checkbox("Texture dumping", &host.cfg.dump_textures);
```

Comment:
im not a very big fan of this change, if you stand by this change then the other elements like "Log exports" should turn into "Exports logging". Also if you have time you can correct the capitalization on that menu too (the debug one, the one that has Log Exports)

Reply:
It looks like I fixed all the capitalization on the whole Debug menu while not touching anything that looks vital to the menu.